.circleci/config.yml

```diff
@@ -223,7 +224,7 @@ workflows:
   security-weekly:
     triggers:
       - schedule:
-          cron: "41 4 * * 1"
+          cron: "0 5 * * 1"
```

Comment:
I think these cron timings should remain as they were? I believe all projects have different times,